webservice/src/main/resources/db/migrations/V3__Create_nps_table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE nps (
```

Comment:
The table name can be net_promoter_score?

Reply:
Também levantei essa dúvida e o @yrachid  me respondeu algo que faz sentido:

> Eu só não vi um problema nessa abreviação por ela ser um termo bem difundido no mercado